lib/stub_ui/painting.dart

```diff
@@ -1762,17 +1762,17 @@ class EngineLayer {
 
 /// A complex, one-dimensional subset of a plane.
 ///
-/// A path consists of a number of subpaths, and a _current point_.
+/// A path consists of a number of sub-paths, and a _current point_.
```

Comment:
We should check the Skia docs on this one and be consistent with them. Subjectively, this looks off to me.

Reply:
Objectively, I can't find a single instance of 'subpath' on Skia.org, but I do find one instance of 'sub-path'. Objection overruled. Proceed, sir.